tests/experimental/datasources/test_pandas_datasource.py

```diff
@@ -132,6 +146,23 @@ def test_data_asset_defined_for_io_read_method(self, method_name: str):
 
         assert type_name in asset_class_names
 
+    @pytest.mark.parametrize("asset_class", PandasDatasource.asset_types)
+    def test_minimal_validation(self, asset_class: Type[_FilesystemDataAsset]):
+        with pytest.raises(pydantic.ValidationError) as exc_info:
```

Comment:
Why does this raise a `ValidationError`? Could we add a comment?

Reply:
I can add a comment, but it's because of the extra "forbidden" keyword arg `foo`